BAZProject/MainMovie/MainMovieInteractor.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  MainMovieInteractor.swift
+//  BAZProject
+//
+//  Created 1030364 on 18/10/22.
+//  Copyright © 2022 ___ORGANIZATIONNAME___. All rights reserved.
+//
+//  Template generated by Juanpe Catalán @JuanpeCMiOS
+//
+
+import UIKit
+
+class MainMovieInteractor: MainMovieInteractorProtocol {
```

Comment:
Recuerda marcar tus clases como `final`

```suggestion
final class MainMovieInteractor: MainMovieInteractorProtocol {
```